### PR TITLE
Add melt weak

### PR DESCRIPTION
**Pull request by tduretz**
1) I have changed the signed of rho: Please make it now negative for liquids to be lighter.
2) You can use `force_melt_weak` and `melt_weak` to force `a` factor in flow law
3) Thermoeleastic contribution to divergence was added

### CODE REVIEW

File: MDLIB/FlowLaws.c
Comment (lcandiot) [inline MDLIB/FlowLaws.c:1]: Is this a flow law factor in addition to the melt fraction or is it a function of phi?

**Review by lcandiot — APPROVED** [review]

File: MDLIB/FlowLaws.c
Comment (tduretz) [reply MDLIB/FlowLaws.c:1]: This is the melt weakening factor from the flow law formulation of Hirth & Kohlstedt, $\exp{(a \phi)}$.
The trick is that most flow laws (crust) this parameter is not determined.
Hence, I added this option such that melt weakening can be "added on top" of default database parameters.

**Review by tgeffroy2 — COMMENTED** [review]
Do we need now to chang ethe signe of "drho" to negative for the change or density with melt?

**Comment by tduretz** [comment]
> Do we need now to chang ethe signe of "drho" to negative for the change or density with melt?

Yes, that the main modification for your setup. If don't change the sign, your melts will be denser than the solids :p

**Pull request merged**